src/networks/models.py

```diff
@@ -158,6 +162,9 @@ class NetworkTypes(models.TextChoices):
         FieldPanel("is_peoplefinder_network"),
         FieldPanel("peoplefinder_network"),
     ]
+
+    promote_panels = NetworksHome.promote_panels
```

Comment:
The reason this pattern is uses is to keep consistency in the inheritance tree, so you would only expect to see a class that the model inherits from being referenced.
`NetworksHome` isn't in the inheritance tree of the `Network` page, so this should just redefine the list like so.

```suggestion
    promote_panels = [
        FieldPanel("useful_links"),
    ]
```

I do like that you're thinking about preventing code duplication!